### PR TITLE
[CodeClean] Fix svace issue

### PR DESCRIPTION
**Pull request by niley7464**
This patch fixes svace issue COMPARE_RESULT_OF_NEW
WGID : 112539-112542

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/extra/nnstreamer_grpc_flatbuf.cc [hunk 1]
```diff
@@ -227,9 +227,6 @@ SyncServiceImplFlatbuf::start_client (std::string address)
 
   /* connect the server */
   client_stub_ = TensorService::NewStub (channel);
-  if (client_stub_.get () == nullptr)
```
Comment (myungjoo) [inline ext/nnstreamer/extra/nnstreamer_grpc_flatbuf.cc:230]:
```
if (client_stub_ == nullptr || client_stub_.get () == nullptr)
```
?

Comment (niley7464) [reply ext/nnstreamer/extra/nnstreamer_grpc_flatbuf.cc:230]:
```
WID:13797032 The pointer returned by operator new can't be null, but it's compared with null at nnstreamer_grpc_flatbuf.cc:230.
  if (client_stub_ == nullptr || client_stub_.get () == nullptr)
```
The same defect is detected

Comment (gichan-jang) [reply ext/nnstreamer/extra/nnstreamer_grpc_flatbuf.cc:230]: How about applying try-catch exception handling?

Comment (niley7464) [reply ext/nnstreamer/extra/nnstreamer_grpc_flatbuf.cc:230]: Thank you for the suggestion!
Updated to use try-catch

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Review by myungjoo — APPROVED** [review]

**Pull request merged**